### DISCUSSION TITLE
Global Filter by tag on react component.

### DISCUSSION THREAD

**Discussion by viktorgusev1984** [likec4/likec4#851] [Ideas]
Hello.

There is an idea that would help us work more with documentation.

I really like predicates in view and try to stick to inserting elements without specifying the final one.

At the same time, it is possible to exclude by tag in the view, but in order to make interactive documentation, I would really like to be able to pass a global filter in the react component for further refinement. I would say that it would be enough to add an additional filter to one tag.
For example

```
<LikeC4View viewId={"index"}  filterTag={"tagA"}/>
```
The reason why not to make different views is that I want to receive changes in all views if a new component or relationships is added.

**Comment by davydkov** [comment]
Hi @viktorgusev1984 

Currently, once a component is generated, the layouts become static. If you need a filter, show or hide elements is possible at this time, but not re-layouting the diagrams.

Does this meet your requirements?

**Reply by viktorgusev1984** [reply]
Hi @davydkov 
Yes, this will cover 90% of cases.

**Comment by davydkov** [comment]
@viktorgusev1984 can you please "test-drive" this feature?
Released in [v1.7.3](https://github.com/likec4/likec4/releases/tag/v1.7.3)

This discussion moved to #872
